grunt/watch.js

```diff
@@ -2,20 +2,13 @@
 
 module.exports = {
     options: {
-        event: [ "changed", "added", "deleted" ]
+        event: [ "changed", "added", "deleted" ],
+        spawn: false
     },
     index: {
         files: "<%= copy.index.src %>",
         tasks: [ "copy:index" ]
     },
-    browserify: {
-        files: [
-            "build/templates.js",
-            "source/modules/**/*.js",
-            "!source/modules/*/tests/**/*.*"
-        ],
-        tasks: [ "browserify:build" ]
-    },
```

Comment:
Adding the `watch: true` option to the `browserify:build` tasks keeps the bundle in memory, so it can respond without having to recompile *all* of the files, just the changed file. It's the change in `grunt/browserify.js`.